src/main/java/io/debezium/connector/yugabytedb/YugabyteDBConnectorTask.java

```diff
@@ -134,17 +134,17 @@ public ChangeEventSourceCoordinator<YBPartition, YugabyteDBOffsetContext> start(
         this.taskContext = new YugabyteDBTaskContext(connectorConfig, schema, topicSelector, taskId);
 
         // Get the tablet ids and load the offsets
-        final Offsets<YBPartition, YugabyteDBOffsetContext> previousOffsets = 
+        final Offsets<YBPartition, YugabyteDBOffsetContext> previousOffsets =
             getPreviousOffsetsFromProviderAndLoader(
                 new YBPartition.Provider(connectorConfig),
                 new YugabyteDBOffsetContext.Loader(connectorConfig));
         final Clock clock = Clock.system();
 
-        YugabyteDBOffsetContext context = new YugabyteDBOffsetContext(previousOffsets, 
+        YugabyteDBOffsetContext context = new YugabyteDBOffsetContext(previousOffsets,
                                                                       connectorConfig);
 
         LoggingContext.PreviousContext previousContext = taskContext
-                .configureLoggingContext(CONTEXT_NAME);
+                .configureLoggingContext(CONTEXT_NAME + "_" + taskId);
```

Comment:
This may change the way we publish metrics as well maybe since we use the task context as one of the keys in forming the name of the metric.

Reply:
But yes, on another thought, the way we form a key is in the pattern:
```
context=streaming,taskId=0
```
and after your change, it might get published as
```
context=streaming_0,taskId=0
```

And the way we detect this metrics is by providing regex like (asterisk is for reference only):
```
context=* taskId=*
```
so ideally we will not be breaking anything.